### PR TITLE
fix: enable `frontend-app-learning` by default

### PR DESCRIPTION
**Pull request by ztraboo**
Ran into this issue when trying to access the new `frontend-app-learning` from the `maple.master` devstack install. Used #865 to fix the issue.

microfrontend-issue-mozjpeg

Made `frontend-app-learning` as a default repo to install on devstack.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ztraboo! I've created [OSPR-6380](https://openedx.atlassian.net/browse/OSPR-6380) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

File: repo.sh [hunk 1]
```diff
@@ -33,11 +33,11 @@ repos=(
     "https://github.com/edx/frontend-app-gradebook.git"
     "https://github.com/edx/frontend-app-payment.git"
     "https://github.com/edx/frontend-app-publisher.git"
+    "https://github.com/edx/frontend-app-learning.git"
```
Comment (cmltaWt0) [inline repo.sh]: Super small comment - there is a trailing space at the end of the line.

**Review by cmltaWt0 — COMMENTED** [review]
Tested locally - everything is good for me.

**Comment by natabene** [comment]
@ztraboo Thank you for the contribution! Please sign new version of our CLA (or have your employer sign a new version of the entity agreement), and we will be able to review your code.

File: repo.sh (hunk 1 above)
Comment (ztraboo) [reply repo.sh]: @cmltaWt0 I just removed that space. I added it because one of the test failed and I wanted the test to run again.

**Comment by ztraboo** [comment]
>

**Comment by openedx-webhooks** [comment]
@ztraboo Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/devstack&entry.867055334=https://github.com/openedx/devstack/pull/880&entry.1484655318=https://github.com/ztraboo&entry.752974735=2022-01-12+14:10&entry.1917517419=2022-01-12+16:07&entry.2133058324=No) so we can improve your experience in the future.

**Comment by openedx-webhooks** [comment]
Hello @ztraboo: We are unable to continue with review of your submission at this time. Please see the associated JIRA ticket for more explanation.

**Comment by ztraboo** [comment]
@natabene I accidentally closed out then reopened this PR. Can you re-open this on your end? Also I waiting to get my Entity CLA agreement signed.

**Comment by cmltaWt0** [comment]
@ztraboo I think it will be better to open two separate PRs.
1. Backport of https://github.com/openedx/devstack/commit/19fdd17e5c8ed1bff891d07064b078db372c0369
2. Moving learning app to release section.

In case CLA signing process delays backporting I can do it on my own.

**Comment by ztraboo** [comment]
@cmltaWt0 Sounds good. I'll make those two separate PR that you mention now.

**Comment by ztraboo** [comment]
@cmltaWt0 See #881 and #882 for these two separate PR.

**Pull request closed without merging**